xvkInterface.hpp

```diff
@@ -2990,10 +2990,11 @@ namespace xvk { namespace Interface {
 			return vkGetMemoryWin32HandleKHR(handle, pGetWin32HandleInfo, pHandle);
 		}
 
-		inline VkResult GetMemoryWin32HandlePropertiesKHR (VkExternalMemoryHandleTypeFlagBits handleType, HANDLE handle, VkMemoryWin32HandlePropertiesKHR* pMemoryWin32HandleProperties) {
-			XVK_CHECK_FUNC(vkGetMemoryWin32HandlePropertiesKHR)
-			return vkGetMemoryWin32HandlePropertiesKHR(handle, handleType, handle, pMemoryWin32HandleProperties);
-		}
```

Comment:
This file is automatically generated using the generator (included in this repository), therefore merging this change is not practical. 
Can you describe the issue ?

Reply:
I got compile error with first (or third) `handle` argument.

Reply:
Okay, I understand the issue, the device handle class member is also a function argument. 
I will make a fix in the generator that will use `this->handle` in the first argument instead. 
Thanks for finding this issue.

Reply:
The fix for this is pushed on the master branch.